src/Platform/PlatformUnix.cpp

```diff
@@ -16,6 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with Falltergeist.  If not, see <http://www.gnu.org/licenses/>.
  */
+#if defined(__unix__) || defined(__APPLE__) || defined(__linux__)
```

Comment:
Does this look necessary? Shouldn't the build system handle the conditional build of the whole file there?